assets/scss/libs/iconfont/_icon-font.scss

```diff
@@ -60,163 +60,167 @@
     content: "\E009";

```

Comment:
We added this file into `.gitignore` in order to not commit this kind of changes, since they generate with styles compilation.